### ISSUE TITLE
Change "peer" to "parent" on stats page

### ISSUE THREAD

**Issue by ldodds** [theodi/pathway#226]

**Comment by ldodds** [comment]
Actually "peer" is the right word here, so I've left it

**Comment by sumikasan** [comment]
Then please change the name of the tab from Parent to Peer

![uploads/74cdf499-1e9e-4e6e-a944-00fcf76625f4/Screen Shot 2015-05-05 at 15.12.31.png](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2F74cdf499-1e9e-4e6e-a944-00fcf76625f4%2FScreen+Shot+2015-05-05+at+15.12.31.png)

**Comment by ldodds** [comment]
These are two different words though? "All in your parent organisation" is correct as that's what we're showing.

For the sub-heading I was trying to describe that we're showing scores for all other organisations within your parent -- but something that wasn't too wordy.

Any ideas?
